src/CommandingOfficers/AWBW/AWBWCommander.java

```diff
@@ -36,10 +36,18 @@ public AWBWCommander(CommanderInfo info, GameScenario.GameRules rules)
   @Override
   public int calculateCombatCharge(UnitDelta minion, UnitDelta enemy, boolean isCounter)
   {
-    if( null != getActiveAbility() )
+    if( minion == null || enemy == null )
```

Comment:
Do we often try to calculate combat charge for combat between nonexistent combatants?

Reply:
I don't think so. That null check has been there a long time, I think.